.gitignore

```diff
@@ -104,6 +104,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
+!test.env
```

Comment:
Make the location for this more specific, I sometimes use `test.env` for storing secrets while I am testing something.